src/vdaf/mastic.rs

```diff
@@ -75,6 +81,16 @@ pub struct MasticAggregationParam {
     require_check_flag: bool,
 }
 
+#[cfg(test)]
+impl MasticAggregationParam {
+    fn new(prefixes: Vec<VidpfInput>, require_check_flag: bool) -> Result<Self, VdafError> {
```

Comment:
Which check? We call this the "weight check" in the spec.

Reply:
Yes, it's the weight check.